### ISSUE TITLE
[Peace and Quiet/Mobs] Quieter Wardens

### ISSUE THREAD

**Issue by Binchus** [BedrockTweaks/Files#284]
- Quieter Sculk Shriekers
- Quieter Sculk Sensors
- Quieter Wardens

"Quieter Wardens" pack does not in fact make Wardens quieter. It doesn't sound like a 95% decrease, it sounds the exact same as vanilla.
Same with other packs for sculk blocks, might just be a v1.21.21 issue but I haven't tested other sounds yet.

Using
 - PC: Windows 10
 - Minecraft Version: 1.21.21

**Additional context**
I just want the warden to be quiet, he's causing a bunch of ruckus in my base, and locking him 64 blocks underground wasn't good enough :(

**Comment by SquatchHunter** [comment]
what sound isnt being dampened?

**Comment by Binchus** [comment]
The Warden's sniffing, groaning, sensor alerted sounds. Pretty much all
sounds the Warden makes

**Comment by Chi-a11y** [comment]
> what sound isnt being dampened?
> 
> edit: think I spotted it, the `"ambient" : "mob.warden.idle",` doesnt have a volume field and thus NSPG would of missed it initially, and then removed it later when clearing out "irrelevant" keys, or thats what I suspect has happened atleast, needs testing

@Binchus was right, I also don't think any of the sounds that the warden makes gets lowered down. I've checked the files and the code (I did the test) before I commented about the issue. The idle of the Warden doesn't seem present in Bedrock's sounds.json files afaik

**Comment by SquatchHunter** [comment]
> The idle of the Warden doesn't seem present in Bedrock's sounds.json files afaik

It is, it is just smaller since it lacks the pitch and volume keys - I made the comment after looking at bedrock samples github, it doesnt exist on ours.

As per why its not working for any sound, ill have a play around and see if maybe some keys were renamed and not updated on the samples page (which wouldnt surprise me)

**Comment by PAPISOP** [comment]
I tried muting all the warden sounds to 0 then it didn't work, then i took the advantage of the new sounds.json content log error and added an invalid sound event to check if the file structure is correct and it was surely correct as it threw an error, so I think this is a minecraft bug

TL;DR: It is a minecraft bug
